src/keysym.rs

```diff
@@ -142,18 +144,19 @@ pub enum Keysym {
     KeypadUp,
     Keypad9,
     KeypadPgUp,
-
-    Unknown(u32),
 }
 
-impl TryFrom<u32> for Keysym {
+impl TryFrom<u32> for KeySym {
     type Error = anyhow::Error;
 
     fn try_from(value: u32) -> Result<Self, Self::Error> {
         match value {
             v if v <= ASCII_MAX => {
-                let ac = v.to_ascii_char().unwrap();
```

Comment:
Would have probably been fine to keep the unwrap, considering you're already testing against `ASCII_MAX`